src/PullRequestReleaseNotes/Providers/GitHubPullRequestProvider.cs

```diff
@@ -40,28 +40,28 @@ public PullRequestDto Get(string commitMessage)
             var pullRequestId = ExtractPullRequestNumber(commitMessage);
             if (pullRequestId == null)
                 return null;
-            var pullRequest = _gitHubClient.PullRequest.Get(_programArgs.GitHubOwner, _programArgs.GitHubRepository, (int) pullRequestId).Result;
-            if (pullRequest == null)
-                return null;
-            if (_programArgs.VerboseOutput)
-                Console.WriteLine($"Found #{pullRequest.Number}: {pullRequest.Title}");
-            return GetPullRequestWithLabels(pullRequest);
+            return GetPullRequestWithLabels(pullRequestId.Value);
```

Comment:
pull request number is also an issue number, https://developer.github.com/v3/issues/
> Note: In the past, pull requests and issues were more closely aligned than they are now. As far as the API is concerned, every pull request is an issue, but not every issue is a pull request.
>
>This endpoint may also return pull requests in the response. If an issue is a pull request, the object will include a pull_request key.

in this doc, https://developer.github.com/v3/pulls/, id is `1` in the response, pull request number is `1347`. But seems when using octokit, you can only access the number, id is not exposed in code.
```[
  {
    "id": 1,
    "url": "https://api.github.com/repos/octocat/Hello-World/pulls/1347",
    "html_url": "https://github.com/octocat/Hello-World/pull/1347",
    "diff_url": "https://github.com/octocat/Hello-World/pull/1347.diff",
    "patch_url": "https://github.com/octocat/Hello-World/pull/1347.patch",
    "issue_url": "https://api.github.com/repos/octocat/Hello-World/issues/1347",
    "commits_url": "https://api.github.com/repos/octocat/Hello-World/pulls/1347/commits",
    "review_comments_url": "https://api.github.com/repos/octocat/Hello-World/pulls/1347/comments",
    "review_comment_url": "https://api.github.com/repos/octocat/Hello-World/pulls/comments/{number}",
    "comments_url": "https://api.github.com/repos/octocat/Hello-World/issues/1347/comments",
    "statuses_url": "https://api.github.com/repos/octocat/Hello-World/statuses/6dcb09b5b57875f334f61aebed695e2e4193db5e",
    "number": 1347,
```